calicoctl/commands/clientmgr/client.go

```diff
@@ -45,7 +47,11 @@ func NewClient(cf string) (*client.Client, error) {
 // otherwise will load from environment variables.
 func LoadClientConfig(cf string) (*api.CalicoAPIConfig, error) {
 	if _, err := os.Stat(cf); err != nil {
-		log.Infof("Config file cannot be read - reading config from environment")
+		if (cf != constants.DefaultConfigPath) {
```

Comment:
Looks like in other error scenarios we do something like this: https://github.com/bcreane/calicoctl/blob/0b8193829d8a332b1c7e7d2711146d746f600fa1/calicoctl/commands/config.go#L110